src/main/webapp/app/shared/components/organisms/application-detail-card/application-detail-card.component.scss

```
@@ -0,0 +1,154 @@
+/* Avatar container */
+.avatar {
+  width: 6rem;
+  height: 6rem;
+  position: relative;
+  border-radius: 9999px;
+  overflow: hidden;
+}
+
+/* Full-size object-fit image */
+.avatar-img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+/* Flex container for name, score, and message */
+.user-info-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  flex: 1;
+}
+
+/* Section title text */
+.section-title {
+  width: 100%;
+  margin-bottom: 0.5rem;
+  margin-left: 7.5rem;
+  font-weight: 700;
+}
+
+/* Generic card wrapper */
+.card {
+  width: 24rem;
+  padding-bottom: 1.5rem;
+  border-radius: 0.5rem;
+  outline: 1px solid var(--p-neutral-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.card-header {
+  width: 100%;
+  height: 3.5rem;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.card-body {
+  width: 100%;
+  flex: 1 1;
+  padding-left: 2rem;
+  padding-right: 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Card row layout */
+.card-row {
+  width: 100%;
+  flex: 1;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card-row-icon {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  gap: 0.625rem;
+}
+
+.icon {
+  color: var(--p-primary-color);
+  width: 1.5rem;
+  height: 1.5rem;
+}
+
+.text-card {
+  width: 100%;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  padding-top: 0.75rem;
+  padding-bottom: 0.75rem;
+  border-radius: 0.5rem;
+  outline: 1px solid var(--p-neutral-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.middle-column {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  height: 100%;
+  gap: 0.625rem;
+}
+
+.links-container {
+  width: 100%;
+  padding-bottom: 1rem;
+  display: inline-flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.link-item {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.625rem;
+}
+
+.main-container {
```

Pls add this to adapt the text color in dark mode:
```suggestion
.main-container {
  color: var(--p-text-primary);
```